pages/docs/couple-your-code/couple-your-code-defining-mesh-connectivity.md

```diff
@@ -61,3 +67,30 @@ if(dim==3)
 [...]
 
 ```
+
+## Changes in v3
+
+Version 3 overhauls the definition of meshes.
+
+Connectivity now consists of explicitly defined elements (via the API) and implicitly defined elements.
+preCICE automatically generates all implicit elements.
+As an example, explicitly defining a triangle ABC via the API guarantees the existence of the implicit edges AB, AC, and BC.
+
+Furthermore, all connectivity is defined via vertex IDs only. There are no more edge IDs to worry about.
+The order of vertices also doesn't matter. Triangles BAC and CAB are considered duplicates.
```

Comment:
I assume this is what "considered duplicates" means:

```suggestion
The order of vertices also does not matter. Triangles BAC and CAB are considered duplicates and preCICE will ignore additional definitions of the same triangle.
```

(somehow the `does not` is supposed to be clearer than `doesn't`)

Reply:
The addition is not true. preCICE has to remove duplicates, which is computationally expensive. We should clarify that this comes at a cost.

```suggestion
The order of vertices also does not matter. Triangles BAC and CAB are considered duplicates and preCICE removes one of them during the deduplication step.
```